Src/BootCamp.Chapter/BalanceParser.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    public class BalanceParser
+    {
+        public Person[] Parser(string[] contents)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'Parser' a static method.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5053435)